examples/with-cookie-auth/components/Layout.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Link from 'next/link'
+import { processLogout } from '../lib/auth'
+
+const Layout = ({ auth, children }) => {
+  const { user = {} } = auth || {}
+  return (
+    <div>
+      <span>Welcome, {user.name || 'Guest'}!</span>
+      <ul style={{ listStyle: 'none', float: 'right' }}>
+        <li><Link href='/'><a>Home</a></Link></li>
+        {user.email
+          ? (
+            <React.Fragment>
+              <li onClick={processLogout} style={{ textDecoration: 'underline', cursor: 'pointer' }}>Logout</li>
```

Comment:
A `<button>` with a click handler should be used instead for proper semantics and better accessibility.